src/lib/adapters/directory-adapter/index.ts

```diff
@@ -1,7 +1,7 @@
 import assert from "node:assert";
 import path from "node:path";
 
-import { deepMerge, filteredData } from "../utils";
+import { deepMerge, filteredData } from "../../utils";
```

Comment:
I moved the utilities one level up, and made it a directory with a barrel file. This way it's easier to find all the different utilities and makes it easer to see where non-exported functions are used.

Reply:
awesome thank you!